swift/ql/lib/codeql/swift/generated/decl/ModuleDecl.qll

```diff
@@ -7,10 +7,16 @@ module Generated {
   class ModuleDecl extends Synth::TModuleDecl, TypeDecl {
     override string getAPrimaryQlClass() { result = "ModuleDecl" }
 
+    /**
+     * Holds if this module is the built-in one.
```

Comment:
Is there only one built-in module?

Reply:
yes :slightly_smiling_face: 

By the way, this is a module you cannot import normally, only if you pass `-parse-std-lib` to the compiler. As the flag suggests, the module should not be used outside the Swift standard library.